packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -1,15 +1,26 @@
+import { URL_INFO } from './project.constants';
+
 export default class {
   /* @ngInject */
-  constructor($http, iceberg) {
+  constructor($http, iceberg, coreConfig) {
     this.project = {};
     this.$http = $http;
     this.iceberg = iceberg;
+    this.coreConfig = coreConfig;
   }
 
   getProjectInfo() {
     return this.project;
   }
 
+  getDocUrl(SUFFIX) {
+    const suffix = URL_INFO[SUFFIX];
+    return `${URL_INFO.PREFIX}/${this.coreConfig
+      .getUserLocale()
+      .toLowerCase()
+      .replace('_', '-')}/${suffix}`;
+  }
```

Comment:
**question:** Why are we using locale? Should not it be subsidiary?

Reply:
yup

Reply:
I have addressed all your comments